src/Form/ChoiceList/ModelChoiceLoader.php

```diff
@@ -111,7 +110,7 @@ public function loadChoiceList($value = null)
                     }
                 }
 
-                $id = implode(AdapterInterface::ID_SEPARATOR, $this->getIdentifierValues($entity));
+                $id = $this->getNormalizedIdentifier($entity);
```

Comment:
Can this be replaced by `$this->modelManager->getNormalizedIdentifier($entity);` directly?